website/project/metadata/osf-preregistration.json

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
Note: this PR will need a migration that adds to OSF Preregistration schema to the database